server/index.js

```diff
@@ -82,9 +82,9 @@ secure.xss()
 .csrf()
 .xframe()
 .csp({
-  frame: [ editorHost, "https://www.youtube.com/embed/JecFOjD9I3k" ],
-  script: [ editorHost ],
-  connection: [ editorHost ]
+  frameSrc: [ editorHost, "https://www.youtube.com/embed/JecFOjD9I3k" ],
```

Comment:
a variable would be fine, It's not going to change often enough to warrant an env var, but only having to change it in one place is a good idea.